server/schedule/operator/builder.go

```diff
@@ -183,99 +235,190 @@ func (b *Builder) SetLightWeight() *Builder {
 	return b
 }
 
+// SetForceTargetLeader marks the step of transferring leader to target is forcible. It is used for grant leader.
+func (b *Builder) SetForceTargetLeader() *Builder {
+	b.forceTargetLeader = true
+	return b
+}
```

Comment:
What about `EnableForceTargetLeader`, `SetForceTargetLeader` usually receive a parameter. And could you give an example in which case we need to use this function?

Reply:
As the comment says `It is used for grant leader`. I'll rename it.